cysync/src/store/hooks/flows/useSendTransaction.ts

```diff
@@ -99,6 +111,13 @@ export const verifyAddress = (address: string, coin: string) => {
     throw new Error(`Cannot find coin details for coin: ${coin}`);
   }
 
+  if (coinDetails.group === CoinGroup.Near) {
+    const regex =
+      coinDetails.validatorNetworkType === 'testnet'
+        ? /^([a-z0-9]{2,56}[-_]?)+\.testnet$|^[a-f0-9]{64}$/
+        : /^([a-z0-9]{2,59}[-_]?)+\.near$|^[a-f0-9]{64}$/;
+    return regex.test(address);
```

Comment:
I tried this with an implicit address, it was blocking the UI.
Simply checking for the implicit first and then custom second made it faster.
Something like this `/^([a-z0-9]{2,56}[-_]?)+\.testnet$|^[a-f0-9]{64}$/`

Reply:
![Screenshot 2022-07-21 at 6 21 15 PM](https://user-images.githubusercontent.com/102788106/180217609-35ecef71-48c3-4c08-b4c3-72db92949628.png)
After I changed this myself, then this happened :/